custom/icds_reports/views.py

```diff
@@ -442,6 +442,10 @@ def query_month_in_range(self, query_month, start_month):
             in_range = False
         return in_range
 
+    @property
+    def bihar_state_id(self):
```

Comment:
This base view I expect to work with non Bihar API as well. So may be you could modify the method to something like 
```
def state_id_from_state_name(self, state_name):
    return SQLLocation.objects.get(name=state_name, location_type__name='state').location_id
```

Reply:
changed